site/docs/components/tabs.mdx

```diff
@@ -0,0 +1,115 @@
+- Navigation tabs are always used for global navigation at the top of the page or inside an offscreen menu on small screens.
+- Section tabs are used for chunking content, allowing movement between content within a single section of a page.
+---
+
+import DosAndDonts from "docs-components/DosAndDonts"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D18760%253A108" allowfullscreen></iframe>
+
+## Options
+
+- Navigation tabs and Section tabs:
+    - Navigation tabs are always used for global navigation at the top of the page or inside an offscreen menu on small screens.
+    - Section tabs are used for chunking content, allowing movement between content within a single section of a page.
+- Vertical tabs and horizontal tabs are used for establishing hierarchy as appropriate for the product area.
+- Default and reversed
+
+## To keep in mind
+
+- Anatomy:
+    - Tab list: This refers to the container holding each of the tab items.
+    - Selected tab: The current, active tab. One tab—and only one tab—is always selected for a given tab list.
+    - Unselected tab: The other available tabs.
+    - Tab item:
+        - A tab item contains a large clickable target area, a label, a selected tab indicator (when the tab is selected), and optional metadata.
+        - Tab link: The large clickable area of the tab item uses a link to navigate to that tab's content.
+        - Tab label: The label describes what content will be displayed. Use 1 short noun or noun phrase.
+        - Selected tab indicator: This contains a selected tab indicator to show that the tab is current.
+        - Optional metadata: You might display 1 item of metadata, such as a [Badge](/components/badge) to bring data forward from the content in that section or a [Tag](/components/tag) to add supplementary information, such as "New".
+    - Tab content: The content that lives "inside" a tab and is revealed when that tab item is selected.
+    - Separator: To separate tabs from content, use whitespace or the edge of another component, such as Title Blocks or Cards, not a [Divider](/components/divider).
+- Styling:
+    - Align navigation tabs and section tabs left (in left-to-right languages) and never center within a page or section.
+    - For small screens, tab items must be presented without loss of information or functionality, and without requiring scrolling in two dimensions for vertical scrolling content at a width equivalent to 320 CSS pixels and horizontal scrolling content at a height equivalent to 256 CSS pixels, as per [W3C: Understanding Success Criterion 1.4.10: Reflow](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html).
+    - Limit tabs to a maximum of 8 tab items and aim for 3–6. Avoid only 2 tabs.
```

Comment:
More likely: "aim for 3-6", a maximum of 6, and for 2 tabs, you might consider if tabs is actually the right choice here.